ui/src/components/FilterField.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { FilterField } from '../components/FilterField';
+import { FilterOption, FilterControl } from '../types';
+
+const defaultProps = {
+  selectedOptionIds: ['signedin'],
+  options: [
+    {
+      value: 'signedin',
+      label: 'User must be Signed In',
+    },
+    {
+      value: 'signedout',
+      label: 'User must be Signed Out',
+    },
+  ] as FilterOption[],
+  control: 'options' as FilterControl,
+  allowMultipe: false,
+  isEditing: false,
+  onFilterUpdated: jest.fn(),
+};
+
+describe('FilterConfig', () => {
+  it('should render all the options provided', () => {
+    const { getByText } = render(
+      <Router>
```

Comment:
It doesn't, we only need to wrap in a <Router> the components that make use of a Router component, such as <Link>. These are mainly the slot/test items within a list, but I had thought it could be useful to wrap every component in a <Router> so the tests don't start failing if we add a <Link> inside the component. But then I thought more about this and changed my mind, best to keep the tests clean and handle any errors that might arise due to that in due course.